src/code-of-conduct.md

```diff
@@ -3,68 +3,83 @@ layout: content.liquid
+### Where
 
-Attendees violating these rules may be asked to leave the conference without a refund at the sole discretion of the conference organizers.
+This Code of Conduct applies to the venue of PyCon India 2024 Conference, pre-event & post-event informal gatherings, related online forums and all conference related events organised by PyCon India.
 
-Thank you for your consideration and help in making PyCon India a welcoming, friendly event for all of us.
+### Who
 
-Long Version
-------------
+This is applicable to speakers, participants, sponsors, volunteers and organisers.
+
+### Be The Best You Can Be
 
-PyCon India is dedicated to providing a harassment-free conference experience for everyone, regardless of age, gender, sexual orientation, physical appearance, disability, race, religion or employment.
+* Be kind, sensitive, understanding and respectful to other attendees.
+* Listen with empathy when someone has a different perspective.
+* Talk to the volunteers if you see something inappropriate or feel something is wrong.
+* All communication should be appropriate for a professional audience, including people from many different backgrounds.
+* Do not insult or put down attendees or engage in any action that violates the open, welcoming and sharing spirit of the conference.
 
-Harassment includes offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion, sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, sustained disruption of talks or other events, inappropriate physical contact, and unwelcome sexual attention. We have zero tolerance on harassment of conference participants in any form, including, but not limited to the activities mentioned here.
+### Please Refrain From
 
-Participants asked to stop any harassing behavior are expected to comply immediately.
+* Offensive verbal comments related to gender, sexual orientation, disability, physical appearance, body size, race, religion.
+* Violent threats or language directed against another person.
+* Swearing, insults or discriminatory/exclusionary/offensive jokes or language.
+* Deliberate intimidation, stalking, following, harassing photography or recording.
+* Sustained disruption of talks or other events.
+* Using sexually explicit or violent material in presentations. Sexual language or imagery is inappropriate for all aspects of the conference, including talks and exhibits.
+* Unwelcome sexual attention or inappropriate physical contact.
+* Advocating for, or encouraging, any of the above behaviour.
+* Repeated harassment of others. In general, if someone asks you to stop, then stop.
+* Avoid use of sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.

```

Comment:
Add a point that when interacting with other people, to not demean anyone and maintain a polite tone towards others.